lab_3_generate_by_ngrams/main.py

```diff
@@ -143,6 +208,16 @@ def _decode(self, corpus: tuple[int, ...]) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(corpus, tuple) or\
+                len(corpus) == 0:
```

Comment:
Please don't check if a container is empty like that! Can you remember a simpler way to check it?

Reply:
Fixed